pex/pex_bootstrapper.py

```diff
@@ -199,9 +213,13 @@ def is_compressed(entry_point):
   return os.path.exists(entry_point) and not os.path.exists(os.path.join(entry_point, PexInfo.PATH))
 
 
-def bootstrap_pex_env(entry_point):
-  """Bootstrap the current runtime environment using a given pex."""
-  pex_info = _bootstrap(entry_point)
+def bootstrap_pex_env(entry_point, include_pexrc=False, extra_vars=None):
+  """
+  Bootstrap the current runtime environment using a given pex. Optionally, pexrc files from
+  'standard' locations can be included - see pex.variables.DEFAULT_PEXRC_LOCATIONS - or
+  runtime-specific overrides (dict or json)
+  """
+  pex_info = _bootstrap(entry_point, include_pexrc=include_pexrc, extra_vars=extra_vars)
 
-  from .environment import PEXEnvironment
+  from pex.environment import PEXEnvironment
```

Comment:
Please do this for all lazy imports in this file or None (ie: revert). I'd prefer the import style remain consistent in the file; otherwise a reasonable question as a reader is "Why is this different? What import mechanic subtlety am I missing?!".

Reply:
Nice catch, that was left in accidentally.  Completely agree re: consistency.